### PR TITLE
Update README.md

### PR DESCRIPTION
**Pull request by miccal**

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -35,7 +35,7 @@ And there we have it. Google Chrome installed with a few quick commands: no clic
 $ open ~/Applications/"Google Chrome.app"
 ```
 
-## Learn More
+## Learn more
```
Comment (vitorgalvao) [inline README.md]: Why are you downcasing titles?

Comment (miccal) [inline README.md]: For consistency.

Comment (vitorgalvao) [inline README.md]: Other documents do the reverse, though, so this is actually inconsistent.

**Comment by miccal** [comment]
Fixed - apologies for my mistake.

**Comment by vitorgalvao** [comment]
No worries. I had to check to confirm. Could you please [squash your commits](http://davidwalsh.name/squash-commits-git)?

**Comment by miccal** [comment]
I am sorry but I do not know how - I have edited this file directly on the website?

**Comment by vitorgalvao** [comment]
Don’t worry, then, I can do it.

**Comment by vitorgalvao** [comment]
Thank you for the contribution. It needed some fixes, so they were made in 02b67ee7b0bd39d9b5c19de22773633a84887d10. Your contribution is still included (and still credited to you), with the appropriate modifications. Please feel free to ask about any of the changes.

**Pull request closed without merging**